eisbuk-admin/src/hooks/useAuthFlow.tsx

```diff
@@ -77,6 +77,8 @@ const useAuthFlow: UseAuthFlow = (fieldErrorMap) => {
           // if error message not defined for a particular error code, fall back to unknown error
           t(AuthErrorMessage[error.code] || AuthErrorMessage.UNKNOWN);
 
+        console.log("Error code", error.code);
```

Comment:
Is this supposed to still be here? Maybe it was supposed to be removed.

Reply:
Yeah...will remove that